test/lib/spdx.js

```diff
@@ -217,13 +217,17 @@ describe('SPDX utility functions', () => {
       'NOASSERTION': 'NOASSERTION',
       'See license': 'NOASSERTION',
       'MIT OR Apache-2.0': 'MIT OR Apache-2.0',
+      'MIT or Apache-2.0': 'NOASSERTION',
```

Comment:
Is this what we expect or want?  Shouldn't operators be case insensitive?

Reply:
According to SPDX specification, operator has to be upper cased, whereas, the license identifiers are case insensitive.  So our code is behaving correctly.  I added the case here as documentation and for my future fix for curation validation.